src/assets/personalInfo/personalInfo.ts

```diff
@@ -1,27 +1,43 @@
 interface IPersonalPhoto {
-  path: string,
-  alt: string
+  path: string;
+  alt: string;
 }
```

Comment:
💭 Off the bat, I'm not a big fan of this text / photo object.  It ties them together unnecessarily, imo. However, I understand why it is that way. I think we can leave it for now... but it rubs me wrong.

Reply:
Out of curiosity, what do you not like here?

And if you were to change it what would you change it to?